### PR TITLE
[FLINK-8475][config][docs] Integrate JM options

### PR DESCRIPTION
**Pull request by zentol**
## What is the purpose of the change

This PR integrates the JobManager `ConfigOptions` into the configuration docs generator.

## Brief change log

* Add missing descriptions to config options (derived from existing description/javadocs)
* integrate jobmanager configuration table into `config.md` and separate job- and taskmanager sections

### CODE REVIEW

File: docs/ops/config.md [hunk 1]
```diff
@@ -322,7 +321,6 @@ The following parameters configure Flink's JobManager and TaskManagers.
 
 - `taskmanager.exit-on-fatal-akka-error`: Whether the TaskManager shall be terminated in case of a fatal Akka error (quarantining event). (DEFAULT: **false**)
 
-- `jobmanager.tdd.offload.minsize`: Maximum size of the `TaskDeploymentDescriptor`'s serialized task and job information to still transmit them via RPC. Larger blobs may be offloaded to the BLOB server. (DEFAULT: **1 KiB**).
```
Comment (aljoscha) [inline docs/ops/config.md]: This setting simply doesn't exist anymore?

Comment (zentol) [reply docs/ops/config.md]: double-checked and yes, I couldn't find a reference anywhere.

**Comment by zentol** [comment]
plenty of missing options are being added by these PRs 😓

**Comment by zentol** [comment]
merging.

**Pull request closed without merging**